lib/beaker/dsl/helpers.rb

```diff
@@ -1340,6 +1340,38 @@ def fact(name, opts = {})
         fact_on(default, name, opts)
       end
 
+      # Snapshot provided host
+      #
+      # Only vagrant hosts are supported
+      # and you must have https://github.com/scalefactory/vagrant-multiprovider-snap installed
+      #
+      # @param host [Host] The host to snapshot
+      # @param name [String] The name to use as the snapshot name
+      def take_snapshot_on(host, name)
+        host.do_take_snapshot(name)
```

Comment:
Yes, beaker-rspec will also need updating to support this new global variable.  I would expose it in the same way we have the 'hosts' and 'options' - see test_case.rb.